python/buildDcrCoadd.py

```diff
@@ -418,6 +436,10 @@ def _build_model_subroutine(self, initial_solution, verbose=True, max_iter=10, m
         Sets self.model as a list of np.ndarrays
         Sets self.weights as a np.ndarray
         """
+        if divergence_threshold is None:
+            divergence_threshold = 1.05
+        if obs_divergence_threshold is None:
+            obs_divergence_threshold = 1.05
```

Comment:
Maybe something a brief explanation of these numbers would be in order? As in why 1.05 and not 1.01?